File: examples/javascript/get_pricing.js
const BASE_URL = "https://wattcoin-production-81a7.up.railway.app";

async function getPricing() {
  try {
    const response = await fetch(`${BASE_URL}/pricing`);
    if (!response.ok) {
      throw new Error(`HTTP error! status: ${response.status}`);
    }
    const data = await response.json();
    console.log(data);
  } catch (error) {
    console.error("Error fetching pricing:", error);
  }
}

getPricing();

File: examples/python/get_pricing.py
import requests

BASE_URL = "https://wattcoin-production-81a7.up.railway.app"

def get_pricing():
    """Fetch current energy pricing information."""
    response = requests.get(f"{BASE_URL}/pricing")
    response.raise_for_status()
    return response.json()

if __name__ == "__main__":
    pricing = get_pricing()
    print(pricing)
